app/code/Magento/StoreGraphQl/etc/graphql/di.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\StoreGraphQl\Model\Resolver\Store\ExtendedStoreConfigDataProvider">
+        <arguments>
+            <argument name="extendedConfigs" xsi:type="array">
+                <!-- Begin Store Configuration Web Default Pages -->
+                <item name="front" xsi:type="string">web/default/front</item>
+                <item name="cms_home_page" xsi:type="string">web/default/cms_home_page</item>
```

Comment:
Such implementation introduces an implicit dependency on CMS and Theme modules. Since both of these modules will require GraphQL coverage I think it's better to move declarations of exporting config path to corresponding modules (CmsGraphQl and ThemeGraphQl)